apps/html-renderer/packages/generic/src/renderHTML.ts

```diff
@@ -0,0 +1,497 @@
+						})
+						.toPNG()
+
+					const tmpFile = path.join(tmpFolder, `img${pad(i, 5)}.png`)
+					recording.writeFilePromises.push(fs.promises.writeFile(tmpFile, buffer))
+				})
+
+				return filename
+			},
+			stopRecording: async () => {
+				if (!recording) throw new Error(`No current recording`)
+				if (recording.stopped) throw new Error(`Recording already stopped`)
+				recording.stopped = true
+
+				win.webContents.endFrameSubscription()
+
+				// Wait for all current file writes to finish:
+				await Promise.all(recording.writeFilePromises)
+
+				let format: string
+				if (recording.fileName.endsWith('.webm')) {
+					format = 'webm'
+				} else if (recording.fileName.endsWith('.mp4')) {
+					format = 'mp4'
+				} else if (recording.fileName.endsWith('.mov')) {
+					format = 'mov'
+				} else {
+					throw new Error(`Unsupported file format: ${recording.fileName}`)
+				}
+
+				// Convert the pngs to a video:
+				await ffmpeg(logger, [
+					'-hide_banner',
+					'-y',
+					'-framerate',
+					'30',
+					'-s',
+					`${width}x${height}`,
+					'-i',
+					`${recording.tmpFolder}/img%05d.png`,
+					'-f',
+					format, // format: webm
+					'-an', // blocks all audio streams
+					'-c:v',
+					'libvpx-vp9', // encoder for video (use VP9)
+					'-auto-alt-ref',
+					'1',
+					escapeFilePath(recording.fileName),
+				])
+
+				await cleanupTemporaryFiles()
+
+				return recording.fileName
+			},
+			cropRecording: async (croppedFilename0: string) => {
+				if (!recording) throw new Error(`No recording`)
+				if (!recording.stopped) throw new Error(`Recording not stopped yet`)
+
+				const croppedFilename = path.join(outputFolder, croppedFilename0)
+				await fs.promises.mkdir(path.dirname(croppedFilename), { recursive: true })
+
+				// Figure out the active bounding box
+				const boundingBox = {
+					x1: Infinity,
+					x2: -Infinity,
+					y1: Infinity,
+					y2: -Infinity,
+				}
+				await ffmpeg(
+					logger,
+					[
+						'-hide_banner',
+						'-i',
+						escapeFilePath(recording.fileName),
+						'-vf',
+						'bbox=min_val=50',
+						'-f',
+						'null',
+						'-',
+					],
+					{
+						onStderr: (data) => {
+							// [Parsed_bbox_0 @ 000002b6f5d474c0] n:25 pts:833 pts_time:0.833 x1:205 x2:236 y1:614 y2:650 w:32 h:37 crop=32:37:205:614 drawbox=205:614:32:37
+							const m = data.match(
+								/Parsed_bbox.*x1:(?<x1>\d+).*x2:(?<x2>\d+).*y1:(?<y1>\d+).*y2:(?<y2>\d+)/
+							)
+							if (m && m.groups) {
+								boundingBox.x1 = Math.min(boundingBox.x1, parseInt(m.groups.x1, 10))
+								boundingBox.x2 = Math.max(boundingBox.x2, parseInt(m.groups.x2, 10))
+								boundingBox.y1 = Math.min(boundingBox.y1, parseInt(m.groups.y1, 10))
+								boundingBox.y2 = Math.max(boundingBox.y2, parseInt(m.groups.y2, 10))
+							}
+						},
+					}
+				)
+
+				if (
+					boundingBox.x1 === Infinity ||
+					boundingBox.x2 === -Infinity ||
+					boundingBox.y1 === Infinity ||
+					boundingBox.y2 === -Infinity
```

Comment:
```suggestion
					!(Number.isFinite(boundingBox.x1) &&
					Number.isFinite(boundingBox.x2) &&
					Number.isFinite(boundingBox.y1) &&
					Number.isFinite(boundingBox.y2))
```

Will also handle any parsing errors that could happen during `parseInt`.